lapis-e2e/test/mostRecentCommonAncestor.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import { basePath, lapisClient } from './common';
+
+describe('The /mostRecentCommonAncestor endpoint', () => {
+  it('should return MRCA with missing nodes if specified', async () => {
+    const result = await lapisClient.postMostRecentCommonAncestor({
+      mostRecentCommonAncestorResponse: {
```

Comment:
the contents do not appear to be passed to the endpoint... its also weird that the openAPI generated library calls this field response when it should be for the request...

Reply:
The naming issue is probably one that you introduced somewhere in LAPIS. The generator in TS doesn't make up names, it takes them from the spec.

Did you debug the TS code here while running the test? That's not difficult and normally already points you to the problem rather quickly.

Reply:
<img width="625" height="689" alt="image" src="https://github.com/user-attachments/assets/b3b9a680-7b36-4e04-a674-74b2cb1d4ddf" />
That also doesn't look right ;)

Reply:
Thanks this helped me find the bug - I think it is now fixed :-)